x-pack/plugins/security_solution/public/management/pages/trusted_apps/service/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HttpStart } from 'kibana/public';
+import { GetTrustedListAppsResponse } from '../../../../../common/endpoint/types/trusted_apps';
+
+export interface TrustedAppsService {
```

Comment:
I've made the argument of the type GetTrustedAppsListRequest. As for exporting interface - I do practice depending on abstraction instead of implementation, so I do use it instead of implementation class. I was even thinking if maybe I should not use class but a factory function that returns object adhering to the interface. Let me know if you prefer that.